### ISSUE TITLE
Ability to create an audio file

### ISSUE THREAD

**Issue by AlfaOmega33** [happyalu/Flite-TTS-Engine-for-Android#64]
Hi! Is possible add an ability to create a .mp4 audio files (or other) directly from the inside of the app ?? 

es.
1) "create voice from text" button; 
2) enter the text; 
3) save in .mp4. 

I think I might be useful, mainly because so you can "compose" phrases most common without write them every time (maybe for deaf and dumb or similar .. no ??)

**Comment by elawed** [comment]
hi are you currently working on any indic voice?

**Comment by happyalu** [comment]
Good idea (option to save audio files). I'll add it to the backlog, but can't promise when it will get done.

@elawed - No current plans personally to release any indic voice. I can not comment about plans that Carnegie Mellon University or  other organizations like hear2read might have.

**Comment by elawed** [comment]
@happyalu  i,m currently stuck in using the app with the hindi indic voice.

As my initial project to build urdu clustergen voice using festvox manual to build indic voice have left me stuck at last parameter as it gives SOID Error .

Kcan you kindly provide me with screen shots step by step to install the app from source in linux.

**Comment by happyalu** [comment]
@elawed As much as I'd like to, I won't be able to build such a tutorial any time soon.

1. The build script and the readme should have the correct steps to build and deploy a working app.

2. For help with building voices using festvox, please contact the festvox mailing list festvox-talk@festvox.org
